experimental/dds/tree2/src/test/shared-tree/fuzz/fuzzEditReducers.ts

```diff
@@ -56,6 +58,43 @@ export const fuzzReducer = withNumberOfEditsCounted(
 			applyTransactionEdit(tree, contents);
 			return state;
 		},
+		reconnect: async (state, operation) => {
+			const { index } = operation;
+			assert(state.containersInfo !== undefined);
+			state.testTreeProvider?.opProcessingController.resumeProcessing();
+			assert(state.testTreeProvider !== undefined);
```

Comment:
Nit: flipping these two lines would let you remove the `?`:

```suggestion
			assert(state.testTreeProvider !== undefined);
			state.testTreeProvider.opProcessingController.resumeProcessing();
```